### DISCUSSION TITLE
Next.js project with headless WordPress (Gutenberg)

### DISCUSSION THREAD

**Discussion by jacobdubail** [tw-in-js/twind#226] [Q&A] [answered]
I need a little help.

I'm currently using twin.macro. I like its api and features. But, I had to disable JIT and purge because the content authors use tailwind classes in WordPress (Gutenberg). 

Will twind help me? Much of my content is client rendered and relies on tailwind classes that populate via graphql requests. I'm currently loading the entire tailwind stylesheet, which is not going to fly in production.

Thank you!

**Comment by sastan** [comment, marked as answer]
twind should be a perfect fit for this kind of use case. twind generates all needed styles during runtime in the browser (ssr is possible as well). You should start with the [shim](https://twind.dev/handbook/the-shim.html#basic-usage) and maybe optimize loading later.

For next.js we have [@twind/next](https://github.com/tw-in-js/use-twind-with/tree/main/packages/next#readme) as integration.
